onnxruntime/core/framework/session_options.h

```diff
@@ -49,7 +49,7 @@ struct SessionOptions {
   int session_log_severity_level = -1;
   int session_log_verbosity_level = 0;  ///< VLOG level if debug build and session_log_severity_level is 0 (VERBOSE).
 
-  unsigned max_num_graph_transformation_steps = 5;  // TODO choose a good default here?
+  unsigned max_num_graph_transformation_steps = 10;  // TODO choose a good default here?
```

Comment:
Since this is for 1 particular model do we need to update the defaults?
Have you run any perf tests on our test data set to see if this affects model load time?
SessionOptions has an option to set the number of steps? Can we just use that? You will need to expose it from all the APIs though...

Reply:
Please see thread.  The only models affected should be those requiring extra steps, and this is affecting more than one model.  

After this is in though, we should be able to see changes in session creation time in our automated perf lab across a wide range of models.